### PR TITLE
jquery verison upd

### PR DESCRIPTION
**Pull request by ikeagold**

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
   "description": "jQuery Split Pane plugin",
   "license": "MIT",
   "dependencies": {
-    "jquery": "1.11.1"
+    "jquery": ">=1.11.1"
```
Comment (shagstrom) [inline bower.json]: I think that I should accept minor updates only. So "~1.11.1" is my suggestion.

Are you using jquery version 2?

Comment (ikeagold) [inline bower.json]: yes 2.1.4 works fine!, also i think we can test it later on 3.0.0 alpha :)
Also pls look at this: https://github.com/shagstrom/split-pane/issues/14
i writed a some messages to your email from http://www.dreamchain.se

Comment (shagstrom) [inline bower.json]: I have updated bower dep to "~1.11.1". Now in version 0.5.1. If you want jquery 2 I suggest creating a branch. My plugin is supporting IE 8. By allowing >= 1.1.1 i would also allow major changes that might brake my plugin.

Comment (ikeagold) [inline bower.json]: @shagstrom  i'm just forgot about ie8 :8ball:, now i understand it, thanks!

**Pull request closed without merging**